api/include/opentelemetry/context/context.h

```diff
@@ -0,0 +1,65 @@
+#ifndef CONTEXT_H_
+#define CONTEXT_H_
```

Comment:
It looks like other opentelemetry-cpp code uses

```suggestion
#pragma once
```

rather than a #ifndef guard.

(e.g. https://github.com/open-telemetry/opentelemetry-cpp/blob/master/api/include/opentelemetry/common/attribute_value.h or https://github.com/open-telemetry/opentelemetry-cpp/blob/master/exporters/otlp/recordable.h)